CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 18.2.2 - [#742](https://github.com/openfisca/openfisca-france/pull/742)
```

Comment:
Only the tests are changed. But CI will complain if I don't bump the version number ?